contracts/apps/AragonApp.sol

```diff
@@ -18,7 +19,7 @@ import "../evmscript/EVMScriptRunner.sol";
 // Unless overriden, this behaviour enforces those contracts to be usable only behind an AppProxy.
 // ReentrancyGuard, EVMScriptRunner, and ACLSyntaxSugar are not directly used by this contract, but
 // are included so that they are automatically usable by subclassing contracts
-contract AragonApp is AppStorage, Autopetrified, VaultRecoverable, ReentrancyGuard, EVMScriptRunner, ACLSyntaxSugar {
+contract AragonApp is IAragonApp, AppStorage, Autopetrified, VaultRecoverable, ReentrancyGuard, EVMScriptRunner, ACLSyntaxSugar {
```

Comment:
I'm a bit concerned with `IAragonApp` and `AppStorage` both declaring essentially the same functions.

If I understand correctly, it's only because of the `supportsInterface()` that we wouldn't want `AppStorage` to inherit `IAragonApp`, and I'm dubious of how important it is for `supportsInterface()` to exist by default in every app.

I'm not even sure if it'll be that useful to declare an ERC165 interface ID for `IAragonApp`, but if it is considered useful, I might move the `supportsInterface()` implementation into the `AragonApp` contract instead.

Reply:
I don’t like this option that much, because to me it feels natural that the `supportsInterface` method lives where the rest of the interface is defined, plus we have to remove the `ERC165` dependency from it too, which was nice in my opinion, but as it seems you both agree, I’m changing it.